.api_violations.report

```diff
@@ -0,0 +1,3 @@
+API rule violation: names_match,github.com/ligato/networkservicemesh/pkg/nsm/apis/netmesh,NetworkServiceChannel,NetworkServiceName
```

Comment:
Curious... what is this and how is it being used?

Reply:
https://github.com/ligato/networkservicemesh/pull/281#discussion_r215648280

Reply:
@edwarnicke The tl;dr (see link above for details) is that openapi-gen sees the camelcase as an API definition error. This lets us get past this by indicating we don't want to error on these definition errors. It also makes @sbezverk happy since we keep the camelcase. :)

Reply:
Awesome!